www-src/app/replicator/replicator_calendars.coffee

```diff
@@ -45,10 +49,13 @@ module.exports =
                         @initEventsInPhone body.last_seq, cb
             (cb) => @updateCalendars cb
             (cb) => @syncEventsPhone2Pouch cb
-            (cb) => @syncEventsToCozy cb
             (cb) => @syncEventsFromCozy cb
+            (cb) => async.eachSeries @changesFromCozy, @handleConflict.bind(@), cb
```

Comment:
True, bind is not good coffee style.
Actually, a fat arrow on handleConflict definition **won't work** ( because this file is a mixin, added here : https://github.com/jacquarg/cozy-mobile/blob/development/www-src/app/replicator/main.coffee#L27 ; and in this case, the coffee's trick with fat arrow give something (window ?) useless as @ ...) .
But I could write the bind with a fat arrow, there is more lines, but it may be more idiomatic:
```
(cb) => 
    async.eachSeries @changesFromCozy, (doc, cb2) =>
        @handleConflict doc, cb2
    , cb